cli.js

```diff
@@ -56,8 +79,43 @@ function run(file) {
 			return;
 		}
 
-		require(file);
+		var options = {
+			env: assign({}, process.env, {AVA_FORK: 1}),
+			silent: true
+		};
+
+		files++;
+
+		var babel = join(__dirname, 'babel.js');
+
+		var ps = fork(babel, [file], options);
+
+		ps.stdout.on('data', test);
+		ps.stderr.on('data', test);
```

Comment:
I also thought to use some kind of IPC, but turned out that you can send messages only parent -> fork way, but not the opposite. So in a PR, fork process detects if it is a fork (by checking `process.env.AVA_FORK`) and outputs passed tests to `stdout` and failed ones to `stderr`. If you could recommend a better way of exchanging data between processes, that would be great! I am not a huge fan of the current method either.